networking/install.bash

```diff
@@ -60,6 +60,21 @@ then
     yes | ssh-keygen -t rsa -b 4096 -N "" -f "$ssh_key"
     rm -r ${ssh_controlmasters_dir:?}/* 2>/dev/null # close all connections, to prevent any possible weird behaviour
     ssh-add # Start using the new key
+
+    # prompt for continuing
+    while true
```

Comment:
Maybe the better thing to do is before replacing the SSH key ask for permission to continue :)

Reply:
But do you give people another choice? I would say we force the 4096 length for security.

Reply:
The check that is in the if condition is very restrictive. If someone is using a newer encryption pattern or if they are not using ~/.ssh/id_rsa for accessing git then we are unnecessarily overwriting that file. That's why leave it on the user.

Reply:
But that is also not the case right now. Could you edit it? I don't see the solution directly

Reply:
Yeah. Give me 10 minutes.